### PR TITLE
[pt-br] Add /docs/reference/glossary/cronjob.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term cronjob.

### CODE REVIEW

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

File: content/pt-br/docs/reference/glossary/cronjob.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: CronJob
+id: cronjob
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/cron-jobs/
+short_description: >
+  Uma tarefa repetida (um trabalho) que é executada regularmente.
```
Comment (devlware) [inline content/pt-br/docs/reference/glossary/cronjob.md:7]: @MrErlison What do you think about having something like:
"Gerencia um trabalho ou tarefa que é executada regularmente"?

Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/cronjob.md:7]: @devlware I think this pretty much matches the short description for the English version (that does not mention managing):
https://github.com/kubernetes/website/blob/76af0dca59be7523a6a27c1f01fdc60c3be65e94/content/en/docs/reference/glossary/cronjob.md?plain=1#L6-L7

Comment (devlware) [reply content/pt-br/docs/reference/glossary/cronjob.md:7]: sorry, I checked the website.

**Pull request merged**